beacon_node/network/src/status.rs

```diff
@@ -29,7 +29,18 @@ pub(crate) fn status_message<T: BeaconChainTypes>(beacon_chain: &BeaconChain<T>)
         finalized_checkpoint.root = Hash256::zero();
     }
 
-    let earliest_available_slot = beacon_chain.store.get_anchor_info().oldest_block_slot;
+    // If there is no data column custody info in the db, that indicates that
+    // no recent cgc changes have occurred and no cgc backfill is in progress.
+    let earliest_available_slot = if let Ok(Some(data_column_custody_info)) =
+        beacon_chain.store.get_data_column_custody_info(false)
+    {
+        std::cmp::max(
+            beacon_chain.store.get_anchor_info().oldest_block_slot,
+            data_column_custody_info.earliest_data_column_slot,
```

Comment:
What's the reason for using `max` of these values instead of just returning `earliest_data_column_slot`? we should be able to serve anything within the data availability window?